src/client/app/helpers/utilities.ts

```diff
@@ -165,6 +169,19 @@ export function processLibraries(snippet: ISnippet) {
     }
 }
 
+export const trustedSnippetsKey = 'trusted_snippets';
```

Comment:
This feels more than just a utility to me, esp. since we're going to move more logic here (e.g., for snippet trusting,not just for checking if trusted).  Please isolate into a separate file.